src/ReactiveDomain.Users.Tests/UserStoreTests.cs

```diff
@@ -0,0 +1,500 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ReactiveDomain.Users.Tests
+{
+    public class UserStoreTests
```

Comment:
Is this test class obsolete, or do these test need to be fixed and re-activated?

Reply:
it's on the checklist to be done